src/analytics/controllers/controller.py

```diff
@@ -1,11 +1,70 @@
 from flask import Blueprint, request, jsonify
 import logging
 import datetime as dt
+from app import mongo
+from helpers.clarity_api import ClarityApi
+from bson import json_util, ObjectId
+import json
 
 _logger = logging.getLogger(__name__)
 
 analytics_app = Blueprint('analytics_app', __name__)
 
+
+@analytics_app.route('/api/v1/device/measurements', methods=['GET'])
```

Comment:
Is it possible to move all the clarity partial URLs to one python file say for example "constant.py" and just import it in the controller.py? It will make the routes look prettier :)